### PR TITLE
Officially deprecates this repo

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
-# Subvisual Blog
-[![Build Status](https://semaphoreci.com/api/v1/projects/d4cb3462-8468-4dca-888f-9e311039ce8d/430739/shields_badge.svg)](https://semaphoreci.com/gb/blog-subvisual-co)
-[![Code Climate](https://codeclimate.com/github/subvisual/blog.subvisual.co/badges/gpa.svg)](https://codeclimate.com/github/subvisual/blog.subvisual.co)
-[![Test Coverage](https://codeclimate.com/github/subvisual/blog.subvisual.co/badges/coverage.svg)](https://codeclimate.com/github/subvisual/blog.subvisual.co/coverage)
+# Subvisual Blog (Deprecated)
 
-Our blog. Live @ [https://subvisual.co/blog](https://subvisual.co/blog)
+**Notice: This blog has since been reimplemented as a static page. Visit the new
+repo [here](https://github.com/subvisual/blog)**. Bye bye Rails, its's been
```
Comment (pfac) [inline README.md]: * it has been

**Review by pfac — COMMENTED** [review]
:bell:

File: README.md (hunk 1 above)
Comment (naps62) [inline README.md]: > If you mean it is or it has, then use it’s.

source: any google search result about "Its vs It's"

Comment (pfac) [inline README.md]: 1. I don't see the point in the contraction there, the sentence is not that long;
2. You are actually using both its and it's at the same time...

File: README.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
-# Subvisual Blog
-[![Build Status](https://semaphoreci.com/api/v1/projects/d4cb3462-8468-4dca-888f-9e311039ce8d/430739/shields_badge.svg)](https://semaphoreci.com/gb/blog-subvisual-co)
-[![Code Climate](https://codeclimate.com/github/subvisual/blog.subvisual.co/badges/gpa.svg)](https://codeclimate.com/github/subvisual/blog.subvisual.co)
-[![Test Coverage](https://codeclimate.com/github/subvisual/blog.subvisual.co/badges/coverage.svg)](https://codeclimate.com/github/subvisual/blog.subvisual.co/coverage)
```
Comment (fribmendes) [inline README.md]: We should add [this badge](http://unmaintained.tech/)

**Review by frm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (naps62) [inline README.md]: oh that. I didn't notice the extra `s` to be honest. that I can remove

as for the first point. I don't (or rather, I do not) see where the contradiction is

**Review by frm — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (naps62) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (pfac) [inline README.md]: Contraction, not contradiction. `It's` is the contraction of `it is` or `it has`.

**Review by pfac — APPROVED** [review]
Still don't like that contraction...

File: README.md (hunk 1 above)
Comment (naps62) [inline README.md]: I guess I'm dyslexic today...

anyway, I don't see why contracting or not should be decided based on the length of the sentence. It's more about one's style of writing than anything else

**Pull request merged**